### ISSUE TITLE
Should have an option to turn off toast message

### ISSUE THREAD

**Issue by ductranit** [eneim/toro#344]
Hi @eneim, 
I have a report from my users that sometime they see a toast message, like `Unable to instantiate decoder OMX.exynos.avc.dec` in production release.
When I checked your library code, I found that there are 2 places you used the `toast`:  
[ToroExo](https://github.com/eneim/toro/blob/dev-v3/toro-exoplayer/src/main/java/im/ene/toro/exoplayer/ToroExo.java#L244) & [ExoPlayable](https://github.com/eneim/toro/blob/dev-v3/toro-exoplayer/src/main/java/im/ene/toro/exoplayer/ExoPlayable.java#L112)
My issue could be related to Exoplayer codec issue [here](https://github.com/google/ExoPlayer/issues/1286)
It's really annoying because it only happens on several devices, and I can't reproduce it. 
The problem is that I don't want my users see this error in production. 
Can you please add an option to turn this `toast` off and move this error to a callback, then I can do something with it?
Thank you!

**Comment by eneim** [comment]
@ductranit Your quick fix is actually my suggestion ... [see this comment](https://github.com/eneim/toro/blob/dev-v3/toro-exoplayer/src/main/java/im/ene/toro/exoplayer/ExoPlayable.java#L109). Though having an option for it makes sense, I would like the developer using Toro acknowledge about this. Will consider a better UX for this in next release.

**Comment by ductranit** [comment]
Thank you, having an error callback would make sense.
And my quick fix can't work for [ToroExo.createDrmSessionManager()](https://github.com/eneim/toro/blob/dev-v3/toro-exoplayer/src/main/java/im/ene/toro/exoplayer/ToroExo.java#L244) because this class is final
Luckily my app doesn't use it.

**Comment by eneim** [comment]
@ductranit Why would you need ``ToroExo.createDrmSessionManager()`` ? It is a util method to create the DrmSessionManager :-?

**Comment by ductranit** [comment]
@eneim No, I don't need it. Just make sure if someone else use it should have an option to turn toast off. You can throw an exception on it.
Btw thanks for your contribution and your support. Much appreciated.
